emcc.py

```diff
@@ -397,17 +397,28 @@ def find_output_arg(args):
   outargs = []
   specified_target = None
   use_next = False
+  skip = False
+
   for arg in args:
+    keep_arg = True
     if use_next:
       specified_target = arg
       use_next = False
-      continue
-    if arg == '-o':
+      keep_arg = False
+    elif arg == '-mllvm':
+      skip = True
+    elif skip:
+      skip = False
+    elif arg == '-o':
       use_next = True
+      keep_arg = False
     elif arg.startswith('-o'):
       specified_target = arg[2:]
-    else:
+      keep_arg = False
+
+    if keep_arg:
       outargs.append(arg)
+
```

Comment:
I makes me a little sad that this requires so much extra complexity here.. but I don't see an obvious way to avoid it.

Reply:
I've pushed another version that I feel is more straightforward, dropping the booleans that used to store the state between the loop iterations.